src/librustc_ast/README.md

```diff
@@ -3,7 +3,7 @@ The `rustc_ast` crate contains those things concerned purely with syntax
 lexer, macro expander, and utilities for traversing ASTs.
 
 For more information about how these things work in rustc, see the
-rustc guide:
+rustc dev guide:
```

Comment:
Maybe add an active link here?

Reply:
I think we should, but maybe let's handle that on a separate PR?